GN_annotation_wrapper/GN_annotation_wrapper.py

```diff
@@ -0,0 +1,154 @@
+import os
+import sys
+import subprocess
+import filecmp
+import csv
+import shutil
+import argparse
+
+def split(infile,anfile,unanfile):
+	comment_lines = str()
+	ann_data = str()
+	unann_data = str()
+
+	with open(infile,'r') as file:
+		for line in file:
+			if line.startswith('#'):
+				comment_lines += line
+			elif line.upper().startswith('HUGO_SYMBOL'):
+				header = line
+				cols = line.split("\t")
+				hgvsp_index = cols.index("HGVSp_Short")
+			else:
+				data = line.split("\t")
+				if data[hgvsp_index] == "":
+					unann_data += line
+				else:
+					ann_data += line
+						
+	if unann_data != "":
+		ann_file = open(anfile,"w")
+		ann_file.write(comment_lines)
+		ann_file.write(header)
+		ann_file.write(ann_data)
+		ann_file.close()
+		unann_file = open(unanfile,"w")
+		unann_file.write(comment_lines)
+		unann_file.write(header)
```

Comment:
I'm not sure about this but I had an issue before where the annotator didn't re-annotate because it already found HGVSp_Short column (and assumed the file was already annotated). I don't know if that will happen here - just something to look out for if the script doesn't behave as you expect.